packages/@okta/vuepress-theme-prose/components/Quickstart.vue

```diff
@@ -1,265 +1,87 @@
-        this.currentHash = window.location.hash
-      },
-
-      activeFramework: function () {
-        window.location.hash = '/'+this.activeClient+'/'+this.activeServer+'/'+this.activeFramework
-        this.currentHash = window.location.hash
-      }
-    },
-
-    computed: {
-      defaultClient: function () {
-        return this.$themeConfig.quickstarts.clients.find((client) => {
-          if(client.default && client.default == true) {
-            return client
-          }
-        })
-      },
-
-      defaultServer: function () {
-        return this.$themeConfig.quickstarts.servers.find((server) => {
-          if(server.default && server.default == true) {
-            return server
-          }
-        })
-      },
-
-      defaultServerFramework: function () {
-        let defaultServer = this.$themeConfig.quickstarts.servers.find((server) => {
-          if(server.default && server.default == true) {
-            return server
-          }
-        })
-
-        return defaultServer.frameworks.find((framework) => {
-          if(framework.default && framework.default == true) {
-            return framework
-          }
-        })
-      },
-
-      defaultUriHash: function () {
-        return '/' + this.defaultClient.name + '/' + this.defaultServer.name + '/' + this.defaultServerFramework.name + '/'
-      },
-
-      activeServerFrameworks: function () {
-        if(this.activeServer == null) return []
-        let activeServerFrameworks = this.$themeConfig.quickstarts.servers.find((server) => {
-          return server.name == this.activeServer
-        })
-        return activeServerFrameworks.frameworks
-      }
-    },
-
-    methods: {
-      scrollToAccount: function () {
-        this.scrollTo('#account')
-        this.activeTab = 'account'
-      },
-      scrollToServer: function () {
-        this.scrollTo('#server-setup')
-        this.activeTab = 'server'
-      },
-      scrollToClient: function () {
-        this.scrollTo('#client-setup')
-        this.activeTab = 'client'
-      },
-      scrollTo: function (element) {
-        window.scrollTo(0, document.querySelectorAll(element)[0].offsetTop - 150)
-      },
-
-      checkActiveClient: function () {
-        let found =  this.$themeConfig.quickstarts.clients.find((client) => {
-          if(client.name == this.activeClient) {
-            return true
-          }
-        })
-
-        if (!found) {
-          this.activeClient = this.defaultClient.name
-        }
-      },
-
-      checkActiveServer: function () {
-        let found =  this.$themeConfig.quickstarts.servers.find((server) => {
-          if(server.name == this.activeServer) {
-            return true
-          }
-        })
-
-        if (!found) {
-          this.activeServer = this.defaultServer.name
-        }
+      let redirecturl = this.mappings[window.location.hash]
+      if(redirecturl == undefined) {
```

Comment:
`if( !redirecturl ) { `

Also: Why not `redirectUrl` (capital U)?

Reply:
Actually, this could all be:
```
const redirectUrl = this.mappings[window.location.hash] || '/docs/guides';
window.location.replace(redirectUrl);
```